include/Note.h

```diff
@@ -236,6 +242,21 @@ class LMMS_EXPORT Note : public SerializingObject
 	TimePos m_length;
 	TimePos m_pos;
 	DetuningHelper * m_detuning;
+	static constexpr std::array<const char*,12> m_nameList =
```

Comment:
While it is a "list", I'd rename to `m_nameArray` to avoid confusion with `std::list`.

Reply:
And the prefix should be `s_`, because `static`.

Reply:
Funnily enough this suggestion illustrates why you shouldn't apply it, if the container is changed the name will become outdated. I would suggest something container-independent like `m_noteNames`.